homeassistant/components/homekit/accessories.py

```diff
@@ -608,6 +622,77 @@ def async_update_battery(self, battery_level: Any, battery_charging: Any) -> Non
                 "%s: Updated battery charging to %d", self.entity_id, hk_charging
             )
 
+    def track_linked_entity[T](self, data: LinkedEntityStateTracking[T]) -> None:
```

Comment:
Since we have both sync code and async code in this codebase, lets call this `async_track_linked_entity` so we know its safe to call in the event loop